src/Microsoft.Extensions.SecretManager.Tools/CommandContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.SecretManager.Tools.Internal
+{
+    internal class CommandContext
```

Comment:
I'm OK keeping this internal, though the usual pattern I believe is to keep the classes public, and indicate that they're internal via the namespace.